packages/askar-react-native/package.json

```diff
@@ -41,12 +41,12 @@
   },
   "devDependencies": {
     "react": "18.2.0",
-    "react-native": "0.74.1",
```

Comment:
shouldn't this reflect the tested/dev version of react native? I think it is currently resolving to `0.76.6`.  I don't think this is a blocker. but I think `*` might get inadvertently updated to an unexpected major/minor version

Reply:
this comment was on the wrong side of the diff :D my bad.